### PR TITLE
Handle "-" as stdin/stdout mode 

### PR DESCRIPTION
**Pull request by rishiloyola**
Resolve  TODO with 'bazel-team' tag.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: buildifier/buildifier.go [hunk 1]
```diff
@@ -136,9 +136,7 @@ func main() {
 
 	diff = differ.Find()
 
-	// TODO(bazel-team): Handle "-" as stdin/stdout mode too.
-
-	if len(args) == 0 {
+	if (len(args) == 0) || (len(args) == 1 && args[0] == "-") {
```
Comment (pmbethe09) [inline buildifier/buildifier.go]: you shouldn't need the leftmost '(' ')' as == is higher precedence than ||

Comment (rishiloyola) [inline buildifier/buildifier.go]: Yes, I am aware of that. Usually, I put conditions under the brackets.
Do you want to remove it?

Comment (pmbethe09) [inline buildifier/buildifier.go]: yes, the right most paren clarify the precedence of || and && so are needed, but the leftmost are extraneous and should be removed.

**Comment by pmbethe09** [comment]
Jenkins, test this please.

File: buildifier/buildifier.go (hunk 1 above)
Comment (rishiloyola) [inline buildifier/buildifier.go]: Done.

**Pull request merged**